multispool/spool.go

```diff
@@ -0,0 +1,134 @@
+// spool.go - remote spool operations
+// Copyright (C) 2019  David Stainton.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package multispool
```

Comment:
looks like this branch includes multispool too

Reply:
yes its a work in progress. i'll be pushing the multispool client soon but first i need to fix our client notification api since it's currently totally broken